.gitignore

```diff
@@ -98,26 +98,7 @@ perl6.rc
 /rakudo-debug-j
 /rakudo-eval-server
 /rakudo-jdb-server
-/inst-perl6
-/inst-perl6-debug
-/inst-perl6-m
-/inst-perl6-debug-m
-/inst-perl6.exe
-/inst-perl6-debug.exe
-/inst-perl6-m.exe
-/inst-perl6-debug-m.exe
-/inst-perl6w.exe
-/inst-perl6w-m.exe
-/inst-rakudo
-/inst-rakudo-debug
-/inst-rakudo-m
-/inst-rakudo-debug-m
-/inst-rakudo.exe
-/inst-rakudo-debug.exe
-/inst-rakudo-m.exe
-/inst-rakudo-debug-m.exe
-/inst-rakudow.exe
-/inst-rakudow-m.exe
+/inst-*
```

Comment:
Some food for thought: if somebody leaves a temporary note for themselves in a _inst-as-something.txt_, for example, then unpleasant surprise may await on them. Or a copy of an _inst-_ file could be made for later analysis/comparison.

I have briefly verified our current approach and found that we only use extension-based cleanup which is much safer with regard to possible unintended damage done.

So, my thinking is that we better consider the source dir as a workplace where a dev might do many kind of things while experimenting with a new feature, a fix, or whatever.

Reply:
I think you missed, that this is the .gitignore and not the cleanup rule in the Makefile. If not, can you clarify what unpleasant surprise you have in mind?

Reply:
I did miss it, sorry! Ok, then there is no problem here.